portal-ui/src/screens/Console/Tenants/TenantDetails/hop/Hop.tsx

```diff
@@ -0,0 +1,132 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Fragment, useState } from "react";
+import { createStyles, Theme, withStyles } from "@material-ui/core/styles";
+import { Link } from "react-router-dom";
+import { CircularProgress, IconButton } from "@material-ui/core";
+import RefreshIcon from "@material-ui/icons/Refresh";
+import PageHeader from "../../../Common/PageHeader/PageHeader";
+import { containerForHeader } from "../../../Common/FormComponents/common/styleLibrary";
+import ExitToAppIcon from "@material-ui/icons/ExitToApp";
+import history from "./../../../../../history";
+
+interface IHopSimple {
+  classes: any;
+  match: any;
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    breadcrumLink: {
+      textDecoration: "none",
+      color: "black",
+    },
+    iframeStyle: {
+      border: "0px",
+      position: "absolute",
+      height: "calc(100vh - 77px)",
+      width: "100%",
+    },
+    divContainer: {
+      position: "absolute",
+      left: "0px",
+      top: "77px",
```

Comment:
Please change this to integer values
```suggestion
      left: 0,
      top: 77,
```